doc/adoc/user_guide.adoc

```diff
@@ -260,3 +254,18 @@ a pointer to the respective log file.
   SUSE Linux Enterprise Server documentation.
 * In systems that contain multiple root file systems on different mount points
   only the root file system mounted on `/` (primary system) will be migrated.
+
+=== Public and Private Cloud Specific
+* Migration initiation is only supported via a reboot. The
+  suse-migration-sle15-activation package must be installed.
```

Comment:
I think this is confusing with [installation](https://github.com/SUSE/suse-migration-services/blob/master/doc/adoc/user_guide.adoc#3-installation)  where it says Migration can be invoked without `suse-migration-sle15-activation` package, running `run_migration`

Reply:
I agree suse-migration-sle15-activation is only required if you plan to activate the migration via reboot. I think what you wanted to express is that in the public cloud there is no other way than reboot to activate it because kexec method is not working in all public clouds.

Reply:
I think we can say that more generically and describe that in a table


```
 CSP     |  reboot     |  run-migration
-----------------------------------------
Azure    |  ok         |    !
Google   |  ok         |    ok
EC2      |  ok         |    !
```